extensions/Worker.Extensions.Http/src/DefaultFromBodyConversionFeature.cs

```diff
@@ -56,19 +56,19 @@ internal class DefaultFromBodyConversionFeature : IFromBodyConversionFeature
             }
             else if (targetType == typeof(byte[]))
             {
-                result = ReadBytes(requestData, context.CancellationToken);
+                result = ReadBytes(requestData, context.CancellationToken);  // Why is this using the invocations CancellationToken?
```

Comment:
This usage is questionable. We shouldn't be short circuiting a customer's function invocation during binding when it is cancelled. Glancing over [MVC's input formatting implementation](https://github.com/dotnet/aspnetcore/blob/3e08b597894891f05556dc0bee23da53cb1b3e11/src/Mvc/Mvc.Core/src/Formatters/SystemTextJsonInputFormatter.cs#L66), I don't see them using CancellationToken even when it is available in APIs that support it. They _could_ pass the CT (available in `InputFormatterContext`), but they don't.

I think this reinforces that we shouldn't be using CT here either.

Reply:
Yup I was wondering the same thing, if we're going by the principal that we only propagate, then it doesn't make sense for us to act here, especially with this being the only scenario where we do use the CT to cancel an action we are taking (inconsistent)